simpleclient_logback/src/main/java/io/prometheus/client/logback/InstrumentedAppender.java

```diff
@@ -0,0 +1,84 @@
+package io.prometheus.client.logback;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.Appender;
+import ch.qos.logback.core.UnsynchronizedAppenderBase;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+
+public class InstrumentedAppender extends UnsynchronizedAppenderBase<ILoggingEvent> {
+
+  public static final String COUNTER_NAME = "logback_appender_total";
+
+  private final CollectorRegistry registry;
+
+  private Counter counter;
+  private Counter.Child trace;
+  private Counter.Child debug;
+  private Counter.Child info;
+  private Counter.Child warn;
+  private Counter.Child error;
+
+  /**
+   * Create a new instrumented appender using the default registry.
+   */
+  public InstrumentedAppender() {
+    this(CollectorRegistry.defaultRegistry);
+  }
+
+  /**
+   * Create a new instrumented appender using the given registry.
+   *
+   * @param registry the collector registry
+   */
+  public InstrumentedAppender(CollectorRegistry registry) {
+    this.registry = registry;
+    setName(Appender.class.toString());
+  }
+
+  @Override
+  public void start() {
```

Comment:
If you think about file Appenders etc, they have to be provided as singletons or they would break. Its the framework (log4j and logbakc) to ensure that regardless of usage. They use statics so we don't have to :) 

You'd wouldn't create a static FileOutputStream.. Counter is same thing really. Trust me, it handles it.